### PR TITLE
Update osram.js

### PR DESCRIPTION
**Pull request by Danit2**
This is to use the Pushbuttonevents in Z2M and some changes and to use the OTA.

### CODE REVIEW

File: devices/osram.js [hunk 1]
```diff
@@ -395,14 +395,23 @@ module.exports = [
...
     {
         fingerprint: [{modelID: 'Zigbee 3.0 DALI CONV LI', endpoints: [{ID: 10}, {ID: 25}, {ID: 242}]},
             {modelID: 'Zigbee 3.0 DALI CONV LI\u0000', endpoints: [{ID: 10}, {ID: 25}, {ID: 242}]}],
         model: '4062172044776_2',
         vendor: 'OSRAM',
         description: 'Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (one device and pushbutton)',
-        extend: extend.ledvance.light_onoff_brightness(),
+        fromZigbee: [fz.command_toggle, fz.command_move, fz.command_stop],
+        extend: extend.ledvance.light_onoff_brightness({noConfigure: true}),
+        exposes: [e.action(['toggle', 'brightness_move_up', 'brightness_move_down', 'brightness_stop'])],
```
Comment (Koenkk) [inline devices/osram.js:408]: This device should also expose a light right? If yes then change this to:

```js
fromZigbee: [...extend.ledvance.light_onoff_brightness({noConfigure: true}).fromZigbee, fz.command_toggle, fz.command_move, fz.command_stop],
extend: extend.ledvance.light_onoff_brightness({noConfigure: true}),
exposes: [...extend.ledvance.light_onoff_brightness({noConfigure: true}).exposes, e.action(['toggle', 'brightness_move_up', 'brightness_move_down', 'brightness_stop'])],
```

File: devices/osram.js [hunk 2]
```diff
@@ -437,19 +443,19 @@ module.exports = [
         model: '4062172044776_4',
         vendor: 'OSRAM',
         description: 'Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (with two devices and pushbutton)',
+        fromZigbee: [fz.command_toggle, fz.command_move, fz.command_stop],
```
Comment (Koenkk) [inline devices/osram.js:446]: Same as https://github.com/Koenkk/zigbee-herdsman-converters/pull/4819/files#r1002656705

**Comment by Danit2** [comment]
Ahh, I see.
Many Thanks @Koenkk . I didn't know exactly how to do it with the extend.
Changed it now.

File: devices/osram.js [hunk 3]
```diff
@@ -437,19 +444,20 @@ module.exports = [
...
+        fromZigbee: [...extend.ledvance.light_onoff_brightness({noConfigure: true}).fromZigbee,
+            fz.command_toggle, fz.command_move, fz.command_stop],
         extend: extend.ledvance.light_onoff_brightness({noConfigure: true}),
-        exposes: [e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2')],
+        exposes: [e.action(['toggle', 'brightness_move_up', 'brightness_move_down', 'brightness_stop']),
+            e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2')],
+        ota: ota.zigbeeOTA,
         endpoint: (device) => {
-            return {'l1': 10, 'l2': 11};
+            return {'l1': 10, 'l2': 11, 's1': 25};
```
Comment (Koenkk) [inline devices/osram.js:454]: Why was `'s1': 25` added here?

Comment (Danit2) [reply devices/osram.js:454]: If I don't do this, I get a "toggle_undefined" in the "action".  Unfortunately I don't know how to do this differently.

Comment (Koenkk) [reply devices/osram.js:454]: Can you check if the device still works fine when removing the `meta: {multiEndpoint: true},` line?

Comment (Danit2) [reply devices/osram.js:454]: Yes it still works. I can remove this. What does this function do exactly? I thought this is required?

Comment (Koenkk) [reply devices/osram.js]: Ah wait, it is needed. Can you update the actions to have the `_s1` postfix?

File: devices/osram.js [hunk 4]
```diff
@@ -437,19 +444,20 @@ module.exports = [
         model: '4062172044776_4',
         vendor: 'OSRAM',
         description: 'Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (with two devices and pushbutton)',
+        fromZigbee: [...extend.ledvance.light_onoff_brightness({noConfigure: true}).fromZigbee,
+            fz.command_toggle, fz.command_move, fz.command_stop],
         extend: extend.ledvance.light_onoff_brightness({noConfigure: true}),
-        exposes: [e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2')],
+        exposes: [e.action(['toggle', 'brightness_move_up', 'brightness_move_down', 'brightness_stop']),
```
Comment (Koenkk) [inline devices/osram.js:450]: I expect that the produced action are postfixed with `_s1` right? So not `toggle` but `toggle_s1`

Comment (Danit2) [reply devices/osram.js:450]: Exactly. It becomes a postfix with '_s1'

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**